templates/openshift/com.redhat.viaq-openshift.template.json

```diff
@@ -518,5 +588,5 @@
   "settings": {
     "index.refresh_interval": "5s"
   },
-  "template": "viaq-openshift-v2016.06.21.0-*"
+  "template": "logstash-*"
```

Comment:
no, that's an error. It's going to be smth like: `[operational.*, tenant.*]`, where `tenant` is the common prefix for all the indices of tenants and `operational` is the prefix of the operational index name.